panedrlite/panedrlite/panedr.py

```diff
@@ -403,7 +411,37 @@ def is_frame_magic(data):
     return magic == -7777777
 
 
-def read_edr(path, verbose=False):
+all_energies_type = list[list[float]]
```

Comment:
Personal opinion but I prefer type hints to be in full at the call site where possible. 

I get that here they are too long and complicated to be readable and am happy with your choice but just where possible.